.github/workflows/triage-issues.yaml

```
@@ -0,0 +1,29 @@
+name: Issue Management
+
+on:
+  issues:
+    types: [opened]
+
+jobs:
+  triage_or_add_to_project:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check if the issue creator is a maintainer
+        id: check_maintainer
+        run: |
+            echo "::set-output name=is_maintainer::$(curl -s -H 'Authorization: token ${{ secrets.GITHUB_TOKEN }}' \
+            https://api.github.com/repos/${{ github.repository }}/collaborators/${{ github.event.issue.user.login }}/permission \
+            | jq -r '.permission' | grep -q 'admin' && echo 'true' || echo 'false')"
```

From https://josh-ops.com/posts/gh-auth-login-in-actions/...

```suggestion
      - name: Check if the issue creator is a maintainer
        id: check_maintainer
        run: |
            IS_ADMIN=`gh api /repos/rstudio/py-shiny/collaborators/${{ github.event.issue.user.login }}/permission --jq='.user.permissions.admin'`
            echo "is_maintainer=$IS_ADMIN" >> "$GITHUB_OUTPUT"
```